build_parameters.py

```diff
@@ -305,6 +305,8 @@ def fetch_commit_hash(version_link, board, file):
     for i in commits_and_codes:
         if commits_and_codes[i][0] != 'error':
             commite_and_codes_cleanned[i] = commits_and_codes[i]
+    if len(commite_and_codes_cleanned) == 0:
```

Comment:
cleaned (spelling) - ok I see it's a pre-existing wierdness, but an opportunity to fix it? (and commite)

Reply:
Not really a good time to fix this sort of thing.  The variable needs a completely new name, not just a spelling fix :-)

... and I don't even know if "commite" comes from a language-other-than-English or is just a typo...